tests/h/search/index_test.py

```diff
@@ -150,9 +154,9 @@ def test_you_can_make_aggregations_on_user_raw(self, factories, index, search):
         response = search.execute()
 
         user_bucket_1 = next(bucket for bucket in response.aggregations.user_raw_terms.buckets
-                             if bucket["key"] == "acct:someone@example.com")
+            if bucket["key"] == "acct:someone@example.com")
```

Comment:
The change in indentation here causes a lint error on my system, although not one caught by the `make lint` command. Can you change this line and the one below back to the way it was?